site/source/user-manual/service-guides/vaultwarden/bitwarden-walkthrough.rst

```diff
@@ -0,0 +1,457 @@
+    ===
+
+    .. tabs::
+
+        .. group-tab:: Tor
+
+            Begin by :ref:`setting up Tor <tor-ios>` on your iPhone.
+
+            #. Visit the App Store and download the `Bitwarden app <https://apps.apple.com/us/app/bitwarden-password-manager/id1137397744>`_
+
+            #. Open the Bitwarden app. You'll be greeted with a log-in screen.
+            
+            #. Go to the top left gear icon to enter the settings.  
+            
+            #. Head to the "Interfaces" tab in the Vaultwarden service on your Start9 Server:
+
+                .. figure:: /_static/images/services/vaultwarden/vaultwarden-interfaces.png
+                    :width: 50%
+                    :alt: vaultwarden-interfaces
+
+            #. Copy the Tor address:
+
+                .. figure:: /_static/images/services/vaultwarden/vaultwarden-tor-address.png
+                    :width: 50%
+                    :alt: vaultwarden-tor-address
+
+            #. Now send that address to your phone and paste it into Bitwarden but **before you hit save:**
+
+                .. note:: The Tor address you will have copied will begin with **http** - Please change this to **https** instead of **http**
+            
+            #. Now you can hit save, and you'll be returned to the log-in screen.
+
+            #. Go ahead and tap 'Log In,' enter your credentials, and you'll be able to access your Bitwarden app / Vaultwarden server!
+
+        .. group-tab:: LAN
+
+            We suggest using Tor, however it is possible have a good experience with LAN. Once synced, your app and all your passwords will be cached and available when you are on the go and not connected to your Start9 Server, and you will only need to be on LAN to update any edits to your vault.
+
+            Begin by :ref:`setting up LAN <lan-ios>` on your device.
+
+            #. Visit the App Store and download the `Bitwarden app <https://apps.apple.com/us/app/bitwarden-password-manager/id1137397744>`_
+
+            #. Open the Bitwarden app. You'll be greeted with a log-in screen.
+            
+            #. Go to the top left gear icon to enter the settings.  
+            
+            #. Head to the "Interfaces" tab in the Vaultwarden service on your Start9 Server:
+
+                .. figure:: /_static/images/services/vaultwarden/vaultwarden-interfaces.png
+                    :width: 50%
+                    :alt: vaultwarden-interfaces
+
+            #. Copy the LAN address:
+
+                .. figure:: /_static/images/services/vaultwarden/vaultwarden-lan-address.png
+                    :width: 50%
+                    :alt: vaultwarden-lan-address
+
+            #. Now send that address to your phone and paste it into Bitwarden.
+            
+            #. Now you can hit save, and you'll be returned to the log-in screen.
+
+            #. Go ahead and tap 'Log In,' enter your credentials, and you'll be able to access your Bitwarden app / Vaultwarden server!
+
+Desktop Apps
```

Comment:
Desktop apps are still clients - why is this separated from Client Setup?  I'm pretty sure the finnicky nature is due to System-level LAN and Tor setups and not the clients

Reply:
I have restructured this. They now appear under the "Client Setup" title.

Do you want to remove the complaint about finickiness?

Reply:
I think it's better to improve LAN setup than mention it here, yes